accountinfo.jsx

```diff
@@ -0,0 +1,159 @@
+    const newName = document.getElementById('editName').value;
+    const newEmail = document.getElementById('editEmail').value;
+    const newPassword = document.getElementById('editPassword').value;
+
+    if (newName && newEmail && newPassword) {
+      document.querySelector('.profile-name').textContent = newName;
+
+      closeEditPopup();
+      alert('Profile updated successfully!');
+    } else {
+      alert('Please fill in all fields.');
+    }
+  };
+
+  const toggleMenu = () => {
+    console.log('Toggle Menu Clicked!');
+    const dropdown = document.querySelector('.dropdown-content');
+    const settingsDropdown = document.getElementById('settingsDropdown');
+
+    // Hide settings dropdown if it's visible
+    if (settingsDropdown.style.display === 'block') {
+      settingsDropdown.style.display = 'none';
+    }
+
+    // Toggle the menu dropdown
+    dropdown.style.display = dropdown.style.display === 'block' ? 'none' : 'block';
+  };
+
+  const toggleSettings = () => {
+    const settingsDropdown = document.getElementById('settingsDropdown');
+    const dropdown = document.querySelector('.dropdown-content');
+
+    // Hide menu dropdown if it's visible
+    if (dropdown.style.display === 'block') {
+      dropdown.style.display = 'none';
+    }
+
+    // Toggle the settings dropdown
+    settingsDropdown.style.display = settingsDropdown.style.display === 'block' ? 'none' : 'block';
+  };
+
+  const toggleVisibility = () => {
+    const visibilityLabel = document.getElementById('visibilityLabel');
+    const visibilityToggle = document.getElementById('visibilityToggle');
+
+    visibilityLabel.textContent = visibilityToggle.checked ? 'Private' : 'Public';
+  };
+
+return (
+  <React.Fragment>
+    {/* HTML code */}
+    <header>
+      <div className="profile-header">
+        <div className="profile-info">
+          <div className="profile-name">Username</div>
+          <div className="profile-actions"></div>
+
+          <div className="profile-stats">
+            <div className="stat">Posts<br /><span>0</span></div>
+            <div className="stat">Chirping<br /><span>0</span></div>
+            <div className="stat">Chirpers<br /><span>0</span></div>
+          </div>
+        </div>
+
+        <div className="menu-icon" onClick={toggleMenu}>
+          <div className="bar"></div>
+          <div className="bar"></div>
+          <div className="bar"></div>
+          <div className="dropdown-content">
+            <a href="#">Activity</a>
+            <a href="#">Favorites</a>
+            <a href="#">Insights</a>
+          </div>
+        </div>
+
+        <div className="settings-icon" onClick={toggleSettings}>
+          <i className="fas fa-cog"></i>
+          <div className="settings-dropdown" id="settingsDropdown">
+            <a href="#">Language</a>
+            <a href="#">Accessibility</a>
+            <a href="#">Help</a>
+          </div>
+        </div>
+      </div>
+    </header>
+  </React.Fragment>
```

Comment:
Try moving the </React.fragment> all the way down to line 155 where the last div is so the entire html code is wrapped in the fragment. React Fragment essentially allows you to return multiple components without having to add extra tags. If this doesn't work, I recommend using the <></> instead of the React.fragment and this should get rid of the parent element linter error :)